packages/office-ui-fabric-react/src/components/Announced/Announced.base.tsx

```diff
@@ -1,26 +1,24 @@
 import * as React from 'react';
-import { BaseComponent, DelayedRender, classNamesFunction } from '../../Utilities';
+import { BaseComponent, DelayedRender, classNamesFunction, getNativeProps, divProperties } from '../../Utilities';
 import { IProcessedStyleSet } from '../../Styling';
-import { IAnnouncedProps, IAnnouncedStyles, IAnnouncedStyleProps } from './Announced.types';
+import { IAnnouncedProps, IAnnouncedStyles } from './Announced.types';
 
-const getClassNames = classNamesFunction<IAnnouncedStyleProps, IAnnouncedStyles>();
+const getClassNames = classNamesFunction<{}, IAnnouncedStyles>();
 
 export class AnnouncedBase extends BaseComponent<IAnnouncedProps, {}> {
```

Comment:
```suggestion
export class AnnouncedBase extends React.Component<IAnnouncedProps> {
```

If we don't need `BaseComponent` let's prefer `React.Component`.